src/valkey-benchmark.c

```diff
@@ -1541,14 +1549,15 @@ int parseOptions(int argc, char **argv) {
         " -k <boolean>       1=keep alive 0=reconnect (default 1)\n"
         " -r <keyspacelen>   Use random keys for SET/GET/INCR, random values for SADD,\n"
         "                    random members and scores for ZADD.\n"
-        "                    Using this option the benchmark will expand the string\n"
-        "                    __rand_int__ inside an argument with a 12 digits number in\n"
+        "                    Using this option the benchmark will replace the string\n"
+        "                    __rand_int__ inside an argument with a random 12 digit number in\n"
         "                    the specified range from 0 to keyspacelen-1. The\n"
```

Comment:
This is printed in a terminal which is 80 chars wide by default. It's not about the width of the source code here but the length of the string between "...\n". With this change the line overflows this length.

It's easy to compensate for it though.

```suggestion
        "                    Using this option the benchmark will replace the string\n"
        "                    __rand_int__ inside an argument with a random 12 digit\n"
        "                    number in the specified range from 0 to keyspacelen-1. The\n"
```